docs/classic-ui/install.md

```diff
@@ -0,0 +1,442 @@
+
+[Service]
+Type=simple
+User=plone
+ExecStart=/opt/plone/zeoserver/bin/runzeo
+KillMode=control-group
+TimeoutStartSec=10
+TimeoutStopSec=10
+
+[Install]
+WantedBy=multi-user.target
+```
+
+`plone-client1.service`
+
+```ini
+[Unit]
+Description=Plone client1
+Requires=plone-zeoserver.service
+
+[Service]
+Type=simple
+User=plone
+ExecStart=/opt/plone/venv/bin/runwsgi /opt/plone/client1/etc/zope.ini
+KillMode=control-group
+TimeoutStartSec=10
+TimeoutStopSec=10
+
+[Install]
+WantedBy=multi-user.target
+```
+
+`plone-client2.service`
+
+```ini
+[Unit]
+Description=Plone client2
+Requires=plone-zeoserver.service
+
+[Service]
+Type=simple
+User=plone
+ExecStart=/opt/plone/venv/bin/runwsgi /opt/plone/client2/etc/zope.ini
+KillMode=control-group
+TimeoutStartSec=10
+TimeoutStopSec=10
+
+[Install]
+WantedBy=multi-user.target
+```
+
+
+Save this configurations in /etc/systemd/system `plone-zeoserver.service`, `plone-client1.service` and  `plone-client2.service` and execute `systemctl daemon-reload` for systemd to read it. After that you can use standard `systemctl` commands to `start`, `restart` or `stop` the Plone instance:
+
+for startup the cluster, use:
+
+```
+systemctl start plone-client1.service
+systemctl start plone-client2.service
+```
+
+the zeoserver starts automatically, because the clients have a requirements directive in the service definition `Requires=plone-zeoserver.service`
+
+for a full shutdown of cluster, use
+
+```
+systemctl stop plone-zeoserver.service
+```
+
+the clients stops automatically, because the clients have a requirements directive in the service definition `Requires=plone-zeoserver.service`
+
+to enable the startup of cluster on boot process:
+
+`systemctl enable plone-client1.service`
+
+`systemctl enable plone-client2.service`
+
+to disable the startup of cluster on boot process:
+
+`systemctl disable plone-client1.service`
+
+`systemctl disable plone-client2.service`
+
+(classic-ui-installation-example-plone-systemd)=
+
+
+
+(classic-ui-installation-buildout-label)=
+
+## Quickstart buildout based installation
+
+install notes for linux
+
+requirements:
+
+- python3.10 or greater
+- python venv module
+
+in our example we use python3.12
+
+On debian based systems you can install python with following command
+
+```bash
+sudo apt install python3.12 python3.12-dev python3.12-venv
+```
+
+Select a directory of your choice
+
+```bash
+mkdir -p /opt/plone && cd /opt/plone
+```
+
+Create a virtual environment
+
+```bash
+python3 -m venv .
+```
+
+Activate the virtual environment
+
+```bash
+source ./bin/activate
+```
+
+install requirements
+
+```bash
+pip install -r https://dist.plone.org/release/6-latest/requirements.txt
+```
+
+add a minimal `buildout.cfg` file to your directory
+
+```cfg
+[buildout]
+extends =
+    https://dist.plone.org/release/6-latest/versions.cfg
+
+parts =
+    instance
+
+[instance]
+recipe = plone.recipe.zope2instance
+user = admin:admin
+http-address = 8080
+eggs =
+    Plone
+```
+
```

Comment:
i would add the following command, which creates the bin/buildout script, so venv does need to be used after that anymore.
```
buildout bootstrap
```
then deactivate venv and jut run ./bin/buildout when ever something in the buildout config has changed.

Reply:
@MrTango Are you actually using the `buildout bootstrap` command regularly?  I was just wondering if I would remove this (and the `buildout init` command) in a new major version of `zc.buildout`.